Makefile

```diff
@@ -217,7 +218,7 @@ release-docs:
 
 .PHONY: github-release
 github-release:
-	go vet ./cmd/release/github_release
-	go run ./cmd/release/github_release/main.go \
+	go vet ./cmd/release/gh-release
+	go run ./cmd/release/gh-release/main.go \
 		--branch=$(RELEASE_BRANCH) \
-		--number=$(PROD_RELEASE)
```

Comment:
It was a thing before the major refactor: https://github.com/aws/eks-distro/pull/1242 

I had removed it because I didn't think we needed it, and the original implementation I did of it was bad. It allowed for way to much flexibility and the logic to deal with all this flexibility was a nightmare to navigate through and maintain